### PR TITLE
init: fix script to use gitea api token

### PR DESCRIPTION
**Pull request by jakubgs**
Using username and password fails when 2FA is enabled:
https://docs.gitea.com/development/api-usage#authentication

### CODE REVIEW

File: files/init-orgs.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+HELP_DEFINITION='''
+Script to initiate Gitea Image registry organization
+'''
+HELP_EXAMPLE='''
+Example:
+\n\t./init-regisitry-org.py 
+  --org org1,org2,org3 \
+  --url https://example.com/gitea \
+  --user username --password password
```
Comment (apentori) [inline files/init-orgs.py]: This part can be updated with `--gitea-token someToken`

File: files/init-orgs.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+from argparse import ArgumentParser, RawTextHelpFormatter
+
+HELP_DEFINITION='''
+Script to initiate Gitea Image registry organization
+'''
+HELP_EXAMPLE='''
+Example:
+\n\t./init-regisitry-org.py 
+  --org org1,org2,org3 \
+  --url https://example.com/gitea \
```
Comment (apentori) [inline files/init-orgs.py]: Should be `--gitea-url`

File: files/init-orgs.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+from requests import request, exceptions
+from argparse import ArgumentParser, RawTextHelpFormatter
+
+HELP_DEFINITION='''
+Script to initiate Gitea Image registry organization
+'''
+HELP_EXAMPLE='''
+Example:
+\n\t./init-regisitry-org.py 
+  --org org1,org2,org3 \
```
Comment (apentori) [inline files/init-orgs.py]: Should be `--orgs`

File: files/init-orgs.py [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        if result.status_code == 404:
+            result.raise_for_status()
+        if result.status_code not in [200, 201, 404]:
+            logging.error('Request failed: %s %s (%d)', method, url, result.status_code)
+            logging.error('Error: %s', result.text)
+            result.raise_for_status()
+        return result
+
+    def create_org(self, org, vis='limited'):
+        return self._request('POST', 'api/v1/orgs', {'visibility':vis,'username':org})
```
Comment (apentori) [inline files/init-orgs.py]: if the url is not provided in param, the defualt value already as `api/v1`

**Review by apentori — COMMENTED** [review]
LGTM

File: files/init-orgs.py (hunk 4 above)
Comment (jakubgs) [reply files/init-orgs.py]: Ah, good catch, fixed.

**Review by apentori — APPROVED** [review]

**Pull request merged**